ioctl/client.go

```diff
@@ -186,6 +190,10 @@ func (c *client) GetAliasMap() map[string]string {
 	return aliases
 }
 
+func (c *client) WriteFile(filename string, data []byte, perm os.FileMode) error {
+	return ioutil.WriteFile(filename, data, perm)
+}
```

Comment:
For client, the function should be `func (c *client) WriteConfig(cfg Config) error`, the config file path and the file mode should be defined and used inside of `client`

Reply:
changed.
Now the "cfg Config" is from cmd/config.go, do I need to migrate config.go?